common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4624,4 +4632,37 @@ object CelebornConf extends Logging {
         "Allowed pattern is: `<host1>:<port1>[,<host2>:<port2>]*`")
       .createWithDefaultString(s"<localhost>:19097")
 
+  val WORKER_INTERNAL_PORT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.internal.port")
+      .categories("worker")
+      .doc(
+        "Internal server port on the Worker where the master nodes connect.")
+      .version("0.5.0")
+      .intConf
+      .createWithDefault(0)
```

Comment:
For these new ports, is 0 the correct default value? I noticed that other ports in this conf have rather specific default values.

Reply:
For the corresponding un-secured ports, the default is 0 as well. When it is 0, then the system picks an ephemeral port. For these ports, that is desired.